<launch>

	<node
		pkg = "turtlesim"
		type = "turtlesim_node"
		name = "turtlesim1"
		respawn = "true"
	>
	<remap
		from="turtle1/cmd_vel"
		to="turtle1/new_keyboard"
	/>
	</node>


	<node
		pkg = "turtlesim"
		type = "turtle_teleop_key"
		name = "teleop_key"
		required = "true"
		launch-prefix = "xterm -e"
	/>

	<node
		pkg = "packagename"
		type = "new_keyboard_executable"
		name = "new_keyboard"
	/>

</launch>
